addons/mail/static/src/components/message/message_tests.js

```diff
@@ -677,8 +682,8 @@ QUnit.test('only show messaging seen indicator if authored by me, after last see
     assert.containsN(
         document.body,
         '.o_MessageSeenIndicator_icon',
-        2,
-        "message component should have two checks (V)"
+        1,
+        "message component should have one check (V)"
```

Comment:
would clarify some members have fetched the message but no one has seen it.

Reply:
done